packages/integral_isolates/test/back_pressure_strategy_test.dart

```diff
@@ -8,27 +8,41 @@ int _testFunction(int input) {
   return input;
 }
 
+String _testFunctionStr(String text) {
```

Comment:
```suggestion
String _testFunction(String text) {
```

Reply:
Impossible, as there is already a function called `_testFunction`